### PR TITLE
Update ug

### PR DESCRIPTION
**Pull request by nishcafe**
I still need to add screenshots of everything, and maybe a pic or two of Jarvis the butler at the start and end, but essentially this would be how everything is worded. If yall dont like certain parts just comment to say so 

For the sake of consistency it might be good to have
- at least one v2.0 command
- at least 1 faq

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -19,116 +19,146 @@ endif::[]
...
+Before delving straight into each feature, here are some things to note - in order to get the most
+out of this user guide:
+
+:sectnums!:
 === Command Format
 
-1. Words in `UPPER CASE` are the parameters supplied by the user, while those
-in `lower case` are **key-words** used by the program.
+* Words in `UPPER CASE` are parameters supplied by the user, while those
+in `lower case` are **commands** used `Jarvis`.
```
Comment (gohsnn) [inline docs/UserGuide.adoc]: Do you mean "used by"?

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -216,371 +275,364 @@ clear-task d/12/09/2019
...
 ----
-export csv f/expenditures
+edit-cca 1 t/sport e/tennis racket
+edit-cca 1 t/performingArt
+edit-cca 1 n/Running e/shoes e/water bottle
 ----
 
-NOTE: Jarvis will create a file called `expenditures.csv` in the current directory, containing all expenditures.
+[NOTE]
+`Jarvis` will soon also be able to edit the milestones of your CCAs in v2.0!
```
Comment (gohsnn) [inline docs/UserGuide.adoc]: Can we standardise how the features are labelled v2.0? Currently they differ across planner, cca tracker and finance tracker.

File: docs/UserGuide.adoc (hunk 1 above)
Comment (nishcafe) [reply docs/UserGuide.adoc]: ah 
yes i do

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -216,371 +275,364 @@ clear-task d/12/09/2019
...
-
-As students, we often forget what we have spent on so far, which makes it hard for us to save since we do not know how
-much we are spending. But do not worry, as here's what the Finance Tracker can
-do for you:
-
-==== Add a single-use purchase: `add-paid`
-So you bought another cup of GongCha at UTown? Let's put that in.
+=== Course Planner
+As an all-encompassing application, we thought it natural to incorporate NUSMods into `Jarvis`. `Jarvis` is a student
+life organiser for NUS students by NUS students. We understand how big a role NUSMods plays in all our lives. Through
```
Comment (ryanYtan) [inline docs/UserGuide.adoc]: Probably don't need to justify the existence of the feature.

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -19,116 +19,146 @@ endif::[]
...
-Download the latest `Jarvis` https://www.google.com/[here].
+1. You can download the latest version of `Jarvis` https://www.google.com/[here].
+
+2. Move `Jarvis` to the directory you would like to use as the
+home folder for the application.
+
+3. Run `jarvis.jar` to start the app. You should see the following on starting the application up:
+
+.Jarvis' Graphical User Interface (GUI)
+image::StartUpGUI.png[]
```
Comment (ryanYtan) [inline docs/UserGuide.adoc]: More sample data in the screenshot will be nicer

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -163,22 +196,57 @@ delete-task 3
...
+To get a quick glimpse of all the tasks you have in your Planner that match a certain <<link-attributes, attribute>>,
+simply pull all tasks relating to that attribute.
+
+Format: `pull-task { t/TYPE | #TAG | p/PRIORITY-LEVEL | f/FREQUENCY-LEVEL | d/DATE }`
+
+===== Example
+----
+`pull-task t/todo`
+`pull-task #cs2106`
+----
```
Comment (ryanYtan) [inline docs/UserGuide.adoc]: Backticks in code-blocks aren't required

File: docs/UserGuide.adoc [hunk 6]
```diff
@@ -657,20 +728,33 @@ It is equivalent to the command `undo 1`
...
-* `redo 5` -- This command will redo the 5 latest commands undone by the user,
+
+`redo 5` -- This command will redo the 5 latest commands undone by the user,
 starting from the latest command that was undone.
 
 [NOTE]
-Suppose you have undone a few commands, if you were to enter in new commands,
-you will not be able to redo the commands.
+If you have entered new commands after undoing previous commands, you will not be able to redo the old
+commands.
```
Comment (marcfyk) [inline docs/UserGuide.adoc]: Will add a new section here for `list-history`

**Pull request merged**